osbs/build/build_request.py

```diff
@@ -1071,6 +1091,12 @@ def render(self, validate=True):
                           ('exit_plugins', 'koji_promote', 'koji_ssl_certs'):
                           self.spec.koji_certs_secret.value,
 
+                          ('postbuild_plugins', 'koji_upload', 'koji_ssl_certs_dir'):
+                          self.spec.koji_certs_secret.value,
+
+                          ('exit_plugins', 'koji_import', 'koji_ssl_certs'):
```

Comment:
Is doing this for `koji_import` premature? Development of this plugin has not been started, correct?